### ISSUE TITLE
broken demo

### ISSUE THREAD

**Issue by Giancarlo1974** [vasttian/vue-admin-vuetify#67]
[The demo ](https://vasttian.github.io/vue-admin-vuetify) works halfway
I believe it is due to an IP that is no longer reachable

**Comment by vasttian** [comment]
Everything looks fine from my side. 
Can you give me some screenshots or a detailed description?

**Comment by Giancarlo1974** [comment]
For example...
In console error log:

Mixed Content: The page at 'https://vasttian.github.io/vue-admin-vuetify/dashboard' was loaded over HTTPS, but requested an insecure image 'http://67.218.155.2:8082/cloud.jpg'. This request has been blocked; the content must be served over HTTPS.

**Comment by vasttian** [comment]
Yes, it seems that there are some similar problems, the links to some files are not adjusted correctly, thank you, I will fix them
